mmv1/third_party/terraform/services/alloydb/resource_alloydb_cluster_test.go.erb

```diff
@@ -1295,3 +1296,48 @@ resource "google_compute_network" "default" {
 data "google_project" "project" {}
 `, context)
 }
+
+<% unless version == 'ga' -%>
+func TestAccAlloydbCluster_withPSC(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckAlloydbClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAlloydbCluster_withPSC(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_alloydb_cluster.default", "psc_config.0.psc_enabled", "true"),
+				),
+			},
+			{
+				ResourceName:      "google_alloydb_cluster.default",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+<% end -%>
+
+func testAccAlloydbCluster_withPSC(context map[string]interface{}) string {
```

Comment:
Could this be included in unless verion==ga block?

otherwise this function is never called in GA provider and will cause a compilation error.